### PR TITLE
created smoke directory with adc project

### PR DESCRIPTION
**Pull request by harsh-patell**

### CODE REVIEW

File: smoke/adc/src/main.c [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      uint16_t data = 0;
+      for (uint8_t i = 0; i < SIZEOF_ARRAY(adc_addy); i++) {
+        adc_read_converted(adc_addy[i], &data);
+        LOG_DEBUG("%d%d: %d\n", adc_addy[i].port, adc_addy[i].pin, data);
+      }
+      delay_ms(1000);
+   }
+}
+
+int main() {
```
Comment (mitchellostler) [inline smoke/adc/src/main.c:40]: You need to add an adc_init(ADC_MODE_SINGLE) after the other initializations

**Review by mitchellostler — COMMENTED** [review]
Just the one thing, otherwise looks good!

File: smoke/adc/src/main.c (hunk 1 above)
Comment (harsh-patell) [reply smoke/adc/src/main.c:40]: Added

Comment (harsh-patell) [reply smoke/adc/src/main.c:40]: I'm not sure why CI fails on format and lint? There was no issue when I tested it

**Review by mitchellostler — APPROVED** [review]

**Pull request merged**